zerver/tests/test_openapi.py

```diff
@@ -343,8 +344,6 @@ def get_standardized_argument_type(self, t: Any) -> type | tuple[type, object]:
         elif origin in [list, abc.Sequence]:
             [st] = get_args(t)
             return (list, self.get_standardized_argument_type(st))
-        elif origin in [dict, abc.Mapping]:
-            return dict
```

Comment:
I assume we get to remove this because the non-pydantic system is losing its last Mapping, and `origin == dict` never actually happens, since `get_origin(dict) = None`?

Reply:
Yes, I think we might have to do the same in a few more places in the tests.